<launch>
  <arg name="map_path" default="."/>
  <arg name="map_name" default="map.yaml"/>
  <arg name="scenario_path" default="$(arg map_path)"/>
  <arg name="scenario_name" default="scene.xml"/>

  <node name="ros_maps_to_pedsim" pkg="ros_maps_to_pedsim" type="ros_maps_to_pedsim.py" output="screen">
    <param name="map_path" value="$(arg map_path)" type="string"/>
    <param name="map_name" value="$(arg map_name)" type="string"/>
    <param name="scenario_path" value="$(arg scenario_path)" type="string"/>
    <param name="scenario_name" value="$(arg scenario_name)" type="string"/>
  </node>

</launch>